src/GitHub.TeamFoundation.14/Services/TeamExplorerServices.cs

```diff
@@ -58,6 +63,14 @@ public void ShowPublishSection()
             publish?.Connect();
         }
 
+        public async Task ShowRepositorySettingsRemotesAsync()
+        {
+            var te = serviceProvider.TryGetService<ITeamExplorer>();
+            var page = await NavigateToPageAsync(te, new Guid("96903923-97e0-474a-9346-31a3ba28e6ff"));
+            var remotes = page?.GetSection(new Guid("2e31f317-7144-4316-8aae-a796e4be1fd4"));
```

Comment:
What's the significance of these GUIDs? Would you be interested in putting them in named variables to document this?

Reply:
They're for the `RepositoryProperties` page and the `Remotes` section. I've put them in named fields.